dist/src/main/distro/conf/mvnd.properties

```diff
@@ -86,7 +86,7 @@
 # MVND_MIN_THREADS
 # The minimum number of threads to use when constructing the default {@code -T} parameter for the daemon.
 # This value is ignored if the user passes @{@code -T}, @{@code --threads} or {@code -Dmvnd.threads} on the command
-# line or if he sets {@code mvnd.threads} in {@code ~/.m2/mvnd.properties}.
+# line or if they set {@code mvnd.threads} in {@code ~/.m2/mvnd.properties}.
```

Comment:
I don't think we can use _they_ and _user_ which is singular.  We could use a passive sentence though: `or if {@code mvnd.threads} is set in ...` which would be better because the value could have been set in the git repo and not directly by the user.

Reply:
The use of _they_ in this context is valid; it's called the [singular they](https://en.wikipedia.org/wiki/Singular_they). (But the passive alternative works too :+1:.)

Reply:
I'd rather use the passive one.

Reply:
@gnodet @Stephan202 which you prefer?

Reply:
Right now the text mixes several grammatical forms. How about:
```
# The minimum number of threads to use when constructing the default {@code -T} 
# parameter for the daemon. This value is ignored if @{@code -T}, @{@code
# --threads} or {@code -Dmvnd.threads} is explicitly specified on the command
# line, or if {@code mvnd.threads} is specified in {@code ~/.m2/mvnd.properties}.
```
This avoids any explicit mention of the user.

Reply:
Thanks @Stephan202 changed to your suggestion. I omitted, "explicitly" (in "explicitly specified"), as it sounded unnecessary.

Reply:
@Stephan202 can you change the PR with your last proposal please ?

Reply:
@gnodet I don't have access to this branch; @findepi did apply the change (modulo the word "explicitly", which LGTM).

Reply:
I enabled `Allow edits by maintainers` if this is needed.